### ISSUE TITLE
High (30-40%) idle CPU usage on Windows 8

### ISSUE THREAD

**Issue by robinlinden** [uTox/uTox#1159]

**Comment by GrayHatter** [comment]
@robinlinden can you reproduce this locally?

If so, ditto with toxcore v0.1.4?

**Comment by robinlinden** [comment]
No, and it was reported on a version they got from https://utox.io/ and I'm pretty sure you serve people old Toxcore with those versions.

**Comment by uname2p** [comment]
Too many offline users will cause a small increase in CPU usage（from 0.1% to 2%~4%）.

Operating system: Win10 x64 v2004
uTox version: 0.18.1

**Comment by avoidr** [comment]
@uname2p, how many is too many?
